api/controllers/v1/jobs.ts

```diff
@@ -62,13 +62,22 @@ export const HandleJobs = async (event: any = {}): Promise<any> => {
             // start the task , don't start the process before processing the notification
             const ecsServices = new ECSContainer(c, consoleLogger);
             const res = await ecsServices.execute(jobId);
+            if (res) {
+              await saveTaskId(jobId, res, consoleLogger);
+            }
             consoleLogger.info(jobId, JSON.stringify(res));
             break;
           case JobStatus[JobStatus.inProgress]:
             queueUrl = c.get('jobUpdatesQueueUrl');
             await NotifyBuildProgress(jobId);
             break;
           case JobStatus[JobStatus.failed]:
+            await NotifyBuildSummary(jobId);
+            const taskId = body['taskId'];
+            if (taskId) {
+              await stopECSTask(taskId, consoleLogger);
+            }
+            break;
```

Comment:
I think alternatively, we can use a new job status (like `timedOut`) if we want to prevent conflating the `failed` status. Wasn't sure how precise we wanted to be though

Reply:
I have a slight preference for `timedOut` but it's *very* slight.

Reply:
Created new `timedOut` status to ensure we're handling separately